test/uk/gov/hmcts/contino/AppPipelineConfigTest.groovy

```diff
@@ -42,9 +42,10 @@ class AppPipelineConfigTest extends Specification {
     when:
       dsl.enableSecurityScan()
     then:
-      assertThat(pipelineConfig.securityScan).isTrue()
+      assertThat(pipelineConfig.securityScanUrlExclusions).isEqualTo("")
+      assertThat(pipelineConfig.useFrontendSecurityScan).isEqualTo("false")
```

Comment:
Not sure why it's expecting this to be false - is this even really needed here?

Reply:
I would expect it to be this:
```suggestion
      assertThat(pipelineConfig.useFrontendSecurityScan).isFalse()
```

Reply:
I thought the same but it seems that it's a string rather than a boolean. Setting it to `isFalse()` causes an error

Reply:
I think I've fixed it. Updated the configuration.scanType to use params.scanType if present or default to `""` and updated the GradleBuilder to expect the value of the environment variable to be `frontend`. Previously was set to expect `true` when I was using a boolean value.